MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -0,0 +1,349 @@
+import UIKit
+import Mapbox
+import MapboxDirections
+import Pulley
+import MapboxNavigation
+import SDWebImage
+
+
+class ArrowFillPolyline: MGLPolylineFeature {}
+class ArrowStrokePolyline: ArrowFillPolyline {}
+
+
+class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDelegate {
+    @IBOutlet weak var mapView: MGLMapView!
```

Comment:
Providing actual customization options may be a big task, and it may not make sense to provide them until the UI design is settled. But we could put a computed `mapView` property in RouteViewController that gives the developer read-only access to this object.